### PR TITLE
Update CMakeLists.txt not setting C++11 compiler flag [9212]

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**
Update CMakeLists.txt not setting C++11 compiler flag.
Update Examples README.txt

Signed-off-by: JLBuenoLopez-eProsima <joseluisbueno@eprosima.com>

### CODE REVIEW

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/2493/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/2307/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/2500/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/2532/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/2343/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/2539/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/2543/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/2354/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/2550/)

File: examples/C++/DDS/Keys/README.txt [hunk 1]
```diff
@@ -78,3 +78,6 @@ Other than this application, he Use Case set of examples contains the following
 * Latejoiners: Shows how a Transient-Local Subscriber receives past samples while Volatile starts receiving from the moment of its creation.
 * SampleConfig: Provides a basic Publish-Subscribe example for the three sample configurations specified in section 5.
 
+5. Run example
+-----------------
+To launch this test execute './DDSKeys' (or DDSKeys.exe on windows).
```
Comment (laura-eprosima) [inline examples/C++/DDS/Keys/README.txt:83]: It also allows the execution of publisher and subscriber separately

File: examples/C++/SampleConfig_Events/README.txt [hunk 2]
```diff
@@ -91,11 +91,14 @@ makes decisions based on the temperature distribution. We would group all sensor
...
 
 	Reliability: Reliable. All samples must reach their destination.
-	Durability: Volatile. Since corrective actions are taken as events come, past triggers provide have no use.
+	Durability: Volatile. Since corrective actions are taken as events come, past triggers have no use.
 	History: Keep-Last. No past alarm information is necessary for present-time transmissions.
 	Additional Settings: Reduce heartbeat period, which dictates system response velocity when a sample is lost. A lower heartbeat period equals fast response on data delivery.
 
+6. Run example
+----------------------
+To launch this test execute: './sampleconfig_events' (or DDSSampleConfigEvents.exe on windows).
```
Comment (laura-eprosima) [inline examples/C++/SampleConfig_Events/README.txt:103]:
```suggestion
To launch this test execute: './sampleconfig_events' (or sampleconfig_events.exe on windows).
```

File: examples/C++/SampleConfig_Multimedia/README.txt [hunk 3]
```diff
@@ -91,11 +91,14 @@ makes decisions based on the temperature distribution. We would group all sensor
...
 		
 	Reliability: Reliable. All samples must reach their destination.
-	Durability: Volatile. Since corrective actions are taken as events come, past triggers provide have no use.
+	Durability: Volatile. Since corrective actions are taken as events come, past triggers have no use.
 	History: Keep-Last. No past alarm information is necessary for present-time transmissions.
 	Additional Settings: Reduce heartbeat period, which dictates system response velocity when a sample is lost. A lower heartbeat period equals fast response on data delivery.
 
+6. Run example
+----------------------
+To launch this test execute: './sampleconfig_multimedia' (or DDSSampleConfigMultimedia.exe on windows).
```
Comment (laura-eprosima) [inline examples/C++/SampleConfig_Multimedia/README.txt:103]:
```suggestion
To launch this test execute: './sampleconfig_multimedia' (or sampleconfig_multimedia.exe on windows).
```

**Review by laura-eprosima — CHANGES_REQUESTED** [review]

**Review by laura-eprosima — APPROVED** [review]

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/2641/)
* Linux aarch64[Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux%20aarch64/53/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/2456/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/2651/)

**Pull request merged**